src/traces/scattergl/index.js

```diff
@@ -946,7 +938,7 @@ function styleTextSelection(cd) {
     var opts = Lib.extendFlat({}, baseOpts);
     var i, j;
 
-    if(els && unels) {
+    if(els.length || unels.length) {
```

Comment:
@etpinard Could you elaborate why .AND. condition is replaced by .OR. here?

Reply:
See commit message in https://github.com/plotly/plotly.js/pull/3810/commits/81367f13e3ee5bb6abfeda5aa12d36c182fac466